testdata/repos/standalone/PackageMetadataXmlCheck/InvalidRemoteID/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>maintainer@random.email</email>
```

Comment:
Nit pick: we are trying to increase his presence in pkgcheck, so let's use here `larry@example.org`

Reply:
Done.